utils.go

```diff
@@ -204,29 +205,27 @@ func NewSnapCtl() *CtlCli {
 
 // Config uses snapctl to get a value from a key, or returns error.
 func (cc *CtlCli) Config(key string) (string, error) {
```

Comment:
How about naming this function GetConfig?

Reply:
This PR is only fixing the error handling and not adding that function.

With that said, renaming the function is a good idea in general to be consistent with snapctl, but that is not a backward-compatible change and will break other applications. Also note that the current naming is the recommended idiomatic way of naming getter functions in Go: https://go.dev/doc/effective_go#Getters